### ISSUE TITLE
cloudflare

### ISSUE THREAD

**Issue by swdlo9** [KyranRana/cloudflare-bypass#186]
Hi , i donw know how to use this script for bypass cloudflare UAM , please you can tald me how i can use this code with just a script?

**Comment by KyranRana** [comment]
I am working on a new version of this script. This script will be out for beta testing soon. 
If you want to be added to alpha testing let me know, but for now you will have to wait again.

**Comment by sedklo** [comment]
Yes i want to be added to alpha testing, but it would be possible to have a script for UAM bypass? example:
node UAM.js http://example.com 100
or
node UAM.js http://example.com proxies.txt 100

please i hope you can understand me, because I have a cloudflare UAM website and I want to test my website, thanks
